### ISSUE TITLE
getMeasurements return array of measure names like getDatabaseNames 

### ISSUE THREAD

**Issue by MusicalCreeper01** [node-influx/node-influx#166]
The title says it all - like with `getDatabaseNames`, `getMeasurements`  should really just return a list of the measure names, not the raw query result. 

Can be done the same way as `getDatabaseNames`:
```
this.queryDB('show measurements', function (err, results) {
    if (err) {
      return callback(err, results)
    }
    return callback(err, _.map(results[0].series[0].values, function (dbarray) { return dbarray[0] }))
  })
```

Or if there is a reason for it returning the raw query, I would suggest adding a new function getMeasurementNames() for ease of use.

**Comment by connor4312** [comment]
This has been resolved in 5.x. Sorry for the delay in responding to this, must have slipped through the cracks! Expect an 5.x RC later this month.

**Comment by connor4312** [comment]
These changes have been published in `5.0.0-alpha.1`. You can install this via:

```
npm install --save influx@next
```
